### PR TITLE
add option to `clip()` the screenshot

### PR DESCRIPTION
**Pull request by yurrriq**
Cleaned up #2 and added a better example.

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -22,7 +22,8 @@ var server = http.createServer(function(req, res) {
       .width(1024)
       .height(768)
       .timeout(100)
-      .format('jpeg');
+      .format('jpeg')
+      .clip();
```
Comment (juliangruber) [inline test.js:26]: needs `t.equal(s._clip, true);`

Comment (juliangruber) [inline test.js:26]: the `"obj"` test needs to be extended as well, with `{ clip: true }`

File: index.js [hunk 2]
```diff
@@ -84,6 +84,17 @@ Screenshot.prototype.format = function(format) {
 };
 
 /**
+ * Clip the screenshot to `width` by `height`.
+ *
+ * @return {Screenshot}
+ */
+
+Screenshot.prototype.clip = function() {
+  this._clip = 'clip';
```
Comment (juliangruber) [inline index.js]: the constructor needs `this._clip = false;`, for perf

**Comment by juliangruber** [comment]
thanks and sorry for the delay, published as `0.6.0`

**Comment by yurrriq** [comment]
:+1:

**Pull request merged**